features/shop/cart_with_different_taxes_for _different_zones.feature

```diff
@@ -0,0 +1,48 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
+    I want to be aware of taxes applied on my order
+
+    Background:
+        Given that store is operating on the France channel
+        And default currency is "EUR"
+        And tax rate "EU VAT" with 23% rate belongs to "Taxable Goods" category for "EU" zone
+        And tax rate "No tax" with 0% rate belongs to "Taxable Goods" category for "Rest of World" zone
+        And tax rate "Low VAT" with 5% rate belongs to "Low-taxed Goods"
+        And there is user "john@example.com" identified by "password123"
+        And catalog has a product "PHP T-Shirt" priced at €100.00 with "Taxable Goods" as tax category
+        And catalog has a product "Symfony Mug" priced at €50.00 with "Low-taxed Goods" as tax category
+        And store has free shipping method
+        And store allows paying offline
+        And I am logged in as "john@example.com"
```

Comment:
We need to cover following cases:

* Completely anonymous Visitor
* Visitor who has entered his address in the checkout
* Customer who signed in and has no address
* Customer who signed in and has a default address
* Customer who signed in and filled his address in checkout

Later we would like to improve the detection for anonymous users and preselect their country etc. But let's start from these cases.

WDYT?